packages/mgt-components/src/components/mgt-person/mgt-person.scss

```diff
@@ -9,706 +9,176 @@
-          }
-        }
-
-        .presence-offline::before,
-        .presence-available::before,
-        .presence-oof-available::before,
-        .presence-away::before,
-        .presence-oof-away::before,
-        .presence-dnd::before,
-        .presence-oof-dnd::before,
-        .presence-oof-offline::before {
-          margin-inline-start: calc(#{$avatar-size-3-lines} * 0.07 - var(--user-presence-margin-offset, 2px));
-        }
+    &.noline,
+    &.oneline {
+      .presence-basic {
+        border-width: 1px;
+        position: relative;
+        bottom: calc(#{$person-avatar-size} * 0.12 - 4px);
       }
     }
 
-    &.fourLines,
-    &.vertical {
-      width: $avatar-size-4-lines;
-      height: $avatar-size-4-lines;
+    &.twolines {
+      --person-avatar-size: 40px;
+    }
 
-      & .initials-text,
-      & .contact-icon {
-        font-size: calc(#{$avatar-size-4-lines} * 0.4);
-      }
-      .user-presence {
-        left: calc(var(--avatar-size-4-lines, var(--avatar-size, 72px)) - 22px);
-        top: calc(var(--avatar-size-4-lines, var(--avatar-size, 72px)) - 22px);
-        width: 20px;
-        height: 20px;
-
-        .presence-basic {
-          width: 16px;
-          height: 16px;
-          border-radius: 50%;
-          padding: 0px;
-          font-size: calc(#{$avatar-size-4-lines} * 0.14);
-
-          & .presence-oof-offline svg {
-            width: calc(#{$avatar-size-4-lines} * 0.14);
-            margin-inline-start: calc(#{$avatar-size-4-lines} * 0.07 - 2px);
-          }
-        }
+    &.threelines,
+    &.large {
+      --person-avatar-size: 56px;
+    }
 
-        .presence-offline::before,
-        .presence-available::before,
-        .presence-oof-available::before,
-        .presence-away::before,
-        .presence-oof-away::before,
-        .presence-dnd::before,
-        .presence-oof-dnd::before,
-        .presence-oof-offline::before {
-          margin-inline-start: calc(#{$avatar-size-4-lines} * 0.07 - 2px);
-        }
-      }
+    &.fourlines {
+      --person-avatar-size: 72px;
     }
 
     &.vertical {
-      margin-inline-start: auto;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      --person-avatar-size: 72px;
     }
 
-    .img-wrapper {
-      height: 100%;
-      width: 100%;
-      display: flex;
-      overflow: hidden;
-      border: $avatar-border;
-      border-radius: $avatar-border-radius;
+    .avatar-wrapper {
+      width: $person-avatar-size;
+      height: $person-avatar-size;
       position: relative;
 
-      img {
-        position: absolute;
-        display: block;
-        inset: -100%;
-        margin: auto;
+      img,
+      .initials,
+      .contact-icon {
         height: 100%;
         width: 100%;
-        object-fit: cover;
-        object-position: top;
-        &:focus() {
-          outline: none;
-        }
+        border-radius: 50%;
```

Comment:
Should this use the variable if we are defining one?

Reply:
We're not defining one, yet. Is this something I could add to allow the developer to customize the avatar border radius?

Reply:
Sorry for the slow reply here, what is the person-border-radius that we define on line 12 for?

Reply:
Oh, I see your initial point. Yes, the variable is defined in line 17. I'm cleaning all of that up. Line 12 is the border radius of the entire person component.
![image](https://user-images.githubusercontent.com/8081536/229797772-42603412-e7e0-49b3-a892-03a73cc818b1.png)